addons/analytic/__openerp__.py

```diff
@@ -18,10 +18,10 @@
     'data': [
         'security/analytic_security.xml',
         'security/ir.model.access.csv',
-        'views/analytic_view.xml',
+        'views/analytic_account_views.xml',
     ],
     'demo': [
-        'data/analytic_demo.xml',
+        'data/analytic_tag_demo.xml',
```

Comment:
I think we can keep analytic_demo.xml. If some day we want to add some demo, we won't have to add a new file.